backend/doc/apache.md

```diff
@@ -84,6 +84,18 @@ Enable this configuration file:
 
     sudo ln -s /etc/apache2/sites-available/tracim.conf /etc/apache2/sites-enabled/tracim.conf
 
+#### Permit access to guest download with old direct link url used before Tracim 3.x (old API path is `/api/v2/`)
```

Comment:
```suggestion
#### Allow guest downloads from direct URLs produced before Tracim v3 (the old API path is `/api/v2/`)
```

Reply:
The problem here is:
- old url work (hyperlink not use `/api/v2/`)
- old direct url not work (direct link is in text view)